src/RoatpWorkflow-QnA.sql

```diff
@@ -2295,3 +2295,295 @@ INSERT INTO [dbo].[WorkflowSequences]
            ,1)
 GO
 
+
+
+
+--- APR-719 adding sequence 4, section 1 interactions
+DECLARE @WorkflowId UNIQUEIDENTIFIER
```

Comment:
These variables are already defined on line 1 of this script

Reply:
ah I just reincluded them to be a distinct thing for the story, as previous section ended with the GO.   
I'll integrate them as one call now